### PR TITLE
[README.md] Improve developer experience

### PR DESCRIPTION
**Pull request by theqp**
By making Getting started faster and simpler.

### CODE REVIEW

**Review by gamalielhere — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -52,21 +52,20 @@ To find out how to disclose a vulnerability visit [hackerone.com/myetherwallet](
 1. Make sure you have NodeJS newer than version 8, but older than version 12, and NPM version 6 or greater.
 2. Open terminal
 3. Clone the repo: `git clone git@github.com:MyEtherWallet/MyEtherWallet.git`
-4. run `npm i` to install node packages.
-5. run `npm run build`. If instructed to edit `package.json` to use newly released versions of dependencies, do so and then run `npm update` and `npm run build` again. After this step succeeds, you can stop and use the offline version by opening the index file from the dist folder with your preferred browser.
-6. start with `npm start`. If instructed to edit `package.json` for newly updated dependencies, see previous step.
-7. If `npm start` fails and above the error message it states 'new update found' then the package.json version of the indicated packages needs to be updated to match the versions shown in the notice.
-8. App should be running in `https://localhost:8080`
+4. run `npm run start:ci`
```
Comment (gamalielhere) [inline README.md:55]: this should just be `npm run start`

Comment (theqp) [reply README.md:55]: problems `start` have:

- slow due to running `package-test.js` and other not needed scripts to start the app
- forces the user to edit `package.json`
- modifies files in the repo

Comment (gamalielhere) [reply README.md]: Those extra those methods were moved to `npm run start:ci` per your suggestion. `npm run start` only does `npm run dev` as seen [here]((https://github.com/MyEtherWallet/MyEtherWallet/pull/2714)

Comment (theqp) [reply README.md]: It's the opposite, `npm run dev` do the extra methods, `npm run start:ci` don't.

**Comment by theqp** [comment]
Closing due history rewrite.

**Pull request closed without merging**